python/tests/simplify.py

```diff
@@ -156,6 +158,7 @@ def rewind_node(self, input_id, output_id):
         Remove the mapping for the specified input and output node pair. This is
         done because there are no edges referring to the node.
         """
+        print('rewind_node was called')
```

Comment:
Stray print here.

Reply:
Thanks. These were removed in the latest commit.